app/views/users/userposts.html.erb

```diff
@@ -0,0 +1 @@
+<h1> this is the user Posts page </h1>
```

Comment:
- [ ] Kindly remove this file. The `app/views/users/index.html.erb` file would take care of all users' posts